salt/modules/zypper.py

```diff
@@ -597,7 +597,7 @@ def install(name=None,
             pkgs=None,
             sources=None,
             downloadonly=None,
-            **kwargs):
+            version=None, **kwargs):
```

Comment:
Could we put `**kwargs` on its own line? Follow the pattern in the rest of the kwargs?